mysql-test/suite/innodb/r/instant_alter_charset.result

```diff
@@ -1803,10 +1749,10 @@ unique key b_idx (b(150))
 alter table tmp
 change a a varchar(150) charset utf16 collate utf16_general_ci,
 algorithm=instant;
-ERROR 0A000: ALGORITHM=INSTANT is not supported. Reason: Cannot change column type INPLACE. Try ALGORITHM=COPY
+ERROR 0A000: ALGORITHM=INSTANT is not supported for this operation. Try ALGORITHM=INPLACE
 alter table tmp
 modify b text charset utf16 collate utf16_general_ci,
 algorithm=instant;
-ERROR 0A000: ALGORITHM=INSTANT is not supported. Reason: Cannot change column type INPLACE. Try ALGORITHM=COPY
+ERROR 0A000: ALGORITHM=INSTANT is not supported for this operation. Try ALGORITHM=INPLACE
 drop table tmp;
 drop table fully_incompatible;
```

Comment:
Before dropping the tables, I would suggest dropping the indexes and then demonstrating that the collations can be changed instantly (while keeping the same character encoding).